Sources/Style.swift

```diff
@@ -67,6 +67,12 @@ extension Style {
             stylable[keyPath: target] = value
         }
     }
+
+    public init<V>(set target: ReferenceWritableKeyPath<Stylable, V?>, to value: V) {
+        style = { stylable, theme, options in
+            stylable[keyPath: target] = .some(value)
```

Comment:
Isn't it better to make swift wrap your value? Like this:

```suggestion
            stylable[keyPath: target] = value
```

Reply:
The compiler just gave me an error, when I was trying to write it this way.

Reply:
![image](https://user-images.githubusercontent.com/11465590/55292334-115cdb80-53f2-11e9-9e52-7696f10843db.png)